dev/app-model.js

```diff
@@ -0,0 +1,213 @@
+import {LitElement, html, css} from 'lit';
+import '@polymer/paper-dialog';
+import '@polymer/paper-button/paper-button.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
```

Comment:
Leave a line. We usually group similar imports. And within the group we maintain the ladder structure.

The hierarchy of the imports should be.
1. Native library imports (lit-html/ lit-element)
2.  Third party imports like paper elements vaadin
3. Project specific imports (components, utils,services)

```
import {LitElement, html, css} from 'lit';

import '@polymer/paper-dialog';
import '@polymer/iron-icon/iron-icon.js';
import '@polymer/iron-icons/iron-icons.js';
import '@polymer/paper-input/paper-input.js';
import '@polymer/paper-button/paper-button.js';

```